### DISCUSSION TITLE
Values in dB(A)?

### DISCUSSION THREAD

**Discussion by josueegatica** [Ifsttar/I-Simpa#363] [General]
Hello, does anyone know if it's possible to obtain values in dB(A) in I-SIMPA?

**Comment by josueegatica** [comment]
I refer to the values provided by the noise map.

**Reply by sidway** [reply]
I am also interested in this. Being able to manipulate noise maps would be very helpful for environment acoustics. I compared some results with iNoise, I was pleased to see good convergence between the results.

**Reply by nicolas-f** [reply]
Hi,

As there is no interaction between the frequency bands, you can put your emission values in dB(A) and you noise level at receivers will be in dB(A).

SPPS is designed for room acoustics and does not take account of diffraction on edges, so for outdoor propagation we recommend to use [NoiseModelling](https://github.com/Universite-Gustave-Eiffel/NoiseModelling) instead.
